league-content/web-mobile.md

```diff
@@ -5,10 +5,15 @@ Resources of interest to students and faculty all about web and mobile developme
 ## Talks
 
 ## Code Samples
-
+[A simple clone of the game Wordle using React and tailwind CSS]()
 ## Tools
 
 ## Courseware
+[Learn FlexBox properties on Flexbox Froggy](https://flexboxfroggy.com/)
```

Comment:
hi, I totally agree that these resources are terrific for learning...but we're looking for items created by Ambassadors themselves that lend themselves to learning opportunities. Can you link any interesting content here that you've generated?

Reply:
Oh great, I will make sure to think about that and write some ideas I had in mind and share them as soon as possible, this will be a great incentive for me to complete them.